Easydict/Swift/View/SettingView/Tabs/TabView/AdvancedTab.swift

```diff
@@ -61,6 +61,26 @@ struct AdvancedTab: View {
                     )
                 }
             }
+            Section {
+                Toggle(isOn: $enableHTTPServer) {
+                    AdvancedTabItemView(
+                        color: .green,
+                        systemImage: "network",
+                        labelText: "setting.general.advance.enable_http_server"
+                    )
+                }
+                TextField(text: $httpPort, prompt: Text("8080")) {
```

Comment:
Maybe set a range limit for the http port input?

Reply:
Yes, but it's too much trouble. Currently, there will be a prompt if the port listening fails.

<img width="812" alt="1601" src="https://github.com/user-attachments/assets/f073a0e5-bbeb-4b40-9d63-8e7b55776695">

Reply:
OK, looks fine.